docker/Dockerfile

```diff
@@ -38,6 +38,7 @@ COPY manage.py newrelic.ini version.json /app/
 COPY --from=frontend /app/.build/ /app/.build/
 
 RUN python manage.py collectstatic --noinput
+RUN python -m whitenoise.compress .build
```

Comment:
Is there a reason you'd want to add it here, since it's already in #7219 ?

Reply:
no, that's an accident - whoops!